### PR TITLE
Support for fcm push topics

### PR DESCRIPTION
**Pull request by or-else**
Added support for fcm push topics to send channel notifications.

Also a couple of bugs fixed.

### CODE REVIEW

File: server/topic.go [hunk 1]
```diff
@@ -1858,7 +1857,12 @@ func (t *Topic) replyGetSub(sess *Session, asUid types.Uid, authLevel auth.Level
 	now := types.TimeNow()
 	id := msg.Id
 	incomingReqTs := msg.Timestamp
-	req := msg.Sub.Get.Sub
+	var req *MsgGetOpts
```
[the remark this answers is not in the document]
Comment (or-else) [reply server/topic.go:1860]: It's not in devel.

[the remark this answers is not in the document]
Comment (aforge) [reply server/topic.go:1860]: I meant this: https://github.com/tinode/chat/commit/85156d487e4eb2ca94713eef58fcfd12b4977e0a

[the remark this answers is not in the document]
Comment (or-else) [reply server/topic.go:1860]: I don't understand. 

There was a crash in next (and channel) but not in devel. The crash came with the change to `replyGetSub` signature:
https://github.com/tinode/chat/blob/devel/server/topic.go#L1750
https://github.com/tinode/chat/blob/next/server/topic.go#L1855

[the remark this answers is not in the document]
Comment (or-else) [reply server/topic.go:1860]: All changes in `devel` are merged into `channel`.

**Pull request merged**